### PR TITLE
changes min players for revs to 30 from 20

### PR DESCRIPTION
**Pull request by Tlaltecuhtli**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl:
tweak: min player req for rev is 30 now 
/:cl:
yeah low pop rev is shit

### CODE REVIEW

**Comment by The-clown-known-as-Pepe** [comment]
good change

**Comment by Dorsisdwarf** [comment]
50 seems too high, I'd recommend 40

**Comment by praisenarsie** [comment]
i recommend 30

**Comment by lzimann** [comment]
50 is way too much, make it 35/40

**Comment by Iatots** [comment]
50 is good.

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Small changes. Bump it to 30 for now and we can see about moving it up from there

**Comment by Iatots** [comment]
D:<

**Comment by Cyberboss** [comment]
Title, changelog

File: code/game/gamemodes/revolution/revolution.dm [hunk 1]
```diff
@@ -13,10 +13,10 @@
 	antag_flag = ROLE_REV
 	false_report_weight = 10
 	restricted_jobs = list("Security Officer", "Warden", "Detective", "AI", "Cyborg","Captain", "Head of Personnel", "Head of Security", "Chief Engineer", "Research Director", "Chief Medical Officer")
-	required_players = 20
-	required_enemies = 1
+	required_players = 30
+	required_enemies = 2
 	recommended_enemies = 3
-	enemy_minimum_age = 14
+	enemy_minimum_age = 18
```
Comment (Cyberboss) [inline code/game/gamemodes/revolution/revolution.dm]: Undocumented, and no reason to change it. Teens can be rebellious

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by Tlaltecuhtli** [comment]
30 is bad tho

**Review by Cyberboss — APPROVED** [review]

File: code/game/gamemodes/revolution/revolution.dm (hunk 1 above)
Comment (The-clown-known-as-Pepe) [reply code/game/gamemodes/revolution/revolution.dm]: kek

**Comment by Robustin** [comment]
Yea this needed to change but we already have a problem of no lowpop modes. It's probably one of the main reasons Sybil struggles to hit critical mass for a decent population, last I checked Sybil was over 50% traitortype rounds. 

I'm of the view that even a pretty lame round is worthwhile if it means shaking things up - maybe a Lite Ops mode where the Ops can't declare war but there's a roundstart message warning the crew. Or maybe give them an alternative objective?

**Comment by Maciekkub** [comment]
@Robustin Well what you say is true, but revhead on lowpop is jsut wasted time. Round starts after long loading, ends after 20 mins because revs and almost no heads and no sec around and then everybody gotta do basic things like RnD, SM, xenobio, gearing up again just because revs.

**Comment by KorPhaeron** [comment]
@Robustin war ops already locks itself to ~50 people online, shouldn't be that hard to make a lower pop version of ops

This is a good change OP

**Comment by lzimann** [comment]
well, the number of ops already scale with the pop, but the tc count probably needs a better scaling since i never touched that

**Comment by Buggy123** [comment]
Change it to 100 minimum players for revs, that seems like a good value.

**Comment by DoctoreMad** [comment]
Disabling revs will encourage good moral values

**Comment by The-clown-known-as-Pepe** [comment]
just had two, two minute rev rounds on 30 pop

30 is still too low

**Pull request merged**